Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -713,6 +716,41 @@ public boolean isEcoDisabled() {
         return economyDisabled;
     }
 
+    private boolean buySignUpdatedToWorth = false;
+
+    private boolean _getBuySignUpdatedToWorth() {
+      return config.getBoolean("economy-update-buy-sign-to-worth", false);
+    }
+
+    @Override
+    public boolean isBuySignUpdatedToWorth() {
+      return buySignUpdatedToWorth;
+    }
+
+    private boolean sellSignUpdatedToWorth = false;
+
+    private boolean _getSellSignUpdatedToWorth() {
+      return config.getBoolean("economy-update-sell-sign-to-worth", false);
+    }
+
+    @Override
+    public boolean isSellSignUpdatedToWorth() {
+      return sellSignUpdatedToWorth;
+    }
+
+    private BigDecimal economyBuyMultiplier = new BigDecimal("1.0");
```

Comment:
Don't initialize variables here
```suggestion
    private BigDecimal economyBuyMultiplier;
```